src/xdocs/checks.xml

```diff
@@ -971,11 +990,15 @@
               Checks that type names conform to a specified pattern.</td>
           </tr>
           <tr>
-            <td><a href="config_misc.html#UncommentedMain">UncommentedMain</a></td>
+            <td><a href="checks/misc/uncommentedmain.html#UncommentedMain">UncommentedMain</a></td>
             <td>Detects uncommented <code>main</code> methods.</td>
```

Comment:
Somehow we lost "`main` methods" from UncommentedMain description:  https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/23a9ce7_2023105251/checks/misc/index.html

See live site: https://checkstyle.sourceforge.io/checks.html#:~:text=Detects%20uncommented%20main%20method

Reply:
@nrmancuso Yes, it's an issue with other checks as well. Present in https://github.com/checkstyle/checkstyle/pull/13264 too.

Reply:
@romani 
![Screenshot from 2023-06-21 16-21-48](https://github.com/checkstyle/checkstyle/assets/23459549/da330a98-53f6-4ee1-adc9-87214fa87d5a)
![Screenshot from 2023-06-21 16-22-09](https://github.com/checkstyle/checkstyle/assets/23459549/1d22ff17-6b7a-4b9c-916e-d6c88293d3db)

Reply:
Opened PR with test at https://github.com/checkstyle/checkstyle/pull/13285

Reply:
Fixed descriptions in table.